traitsui/examples/demo/Advanced/Dynamic_EnumEditor_demo.py

```diff
@@ -97,7 +97,7 @@ class OrderMenu(HasPrivateTraits):
     entree = Enum(values='capabilities.available')
 
     # Reference to the restaurant's current entree capabilities:
-    capabilities = Constant(kitchen_capabilities)
+    capabilities = Instance(kitchen_capabilities, args=())
```

Comment:
Shouldn't we be using the class instead of the instance here because we're setting `args=()`? A meaningful alternative would be to return `kitchen_capabilities` in a `_capabilities_default` method.

```suggestion
    capabilities = Instance(KitchenCapabilities, args=())
```